.gitignore

```diff
@@ -2,3 +2,4 @@ pubspec.lock
 packages
 .pub
 .packages
+.idea
```

Comment:
Please remove this. Should be in your `.gitignore_global` file!

Reply:
Done